rider-fsharp/protocol/src/kotlin/model/RdFSharpExternalFormatterModel.kt

```diff
@@ -0,0 +1,72 @@
+package model
+
+import com.jetbrains.rider.model.nova.ide.SolutionModel
+import com.jetbrains.rd.generator.nova.*
+import com.jetbrains.rd.generator.nova.PredefinedType.*
+import com.jetbrains.rd.generator.nova.csharp.CSharp50Generator
+import java.io.File
+
+@Suppress("unused")
+object RdFSharpExternalFormatterModel : Root() {
+
+    private val rdParsingOptions = structdef {
```

Comment:
It seems to miss a lot of info from the parsing options. Something like the following will likely be formatted wrong:
```fsharp
#if DEBUG
let x = 123
#endif
```

Reply:
Name suggestion: `rdFcsParsingOptions`.